src/mon/MonmapMonitor.cc

```diff
@@ -611,6 +611,17 @@ bool MonmapMonitor::prepare_command(MonOpRequestRef op)
       err = -EINVAL;
       goto reply_no_propose;
     }
+    if (monmap.stretch_mode_enabled) {
+      for (const auto &p : loc) {
```

Comment:
There's a TODO comment several lines up that should be removed if this actually works.

Reply:
Okay will remove that